src/tetris/Square.java

```diff
@@ -0,0 +1,43 @@
+package tetris;
+
+import edu.macalester.graphics.Rectangle;
+
+public class Square {
```

Comment:
I wonder if Board could use this helpful class instead of working with raw booleans and Rectangles?